ocaml/xapi/test_vdi_cbt.ml

```diff
@@ -62,9 +62,95 @@ let test_set_metadata_of_pool_doesnt_allow_cbt_metadata_vdi () =
     Api_errors.(Server_error (vdi_incompatible_type, [Ref.string_of self; Record_util.vdi_type_to_string `cbt_metadata]))
     (fun () -> Xapi_vdi.set_metadata_of_pool ~__context ~self ~value:pool)
 
+let test_get_nbd_info =
+  let make_host __context sR ?(pbd_attached=true) pif ?(pif_attached=true) () =
+    let host = Test_common.make_host ~__context () in
+    let _: _ API.Ref.t = Test_common.make_pbd ~__context ~host ~sR ~currently_attached:pbd_attached () in
+    begin match pif with
+    | Some (network, iP) ->
+      let self = Test_common.make_pif ~__context ~network ~host ~iP () in
+      Db.PIF.set_currently_attached ~__context ~self ~value:pif_attached
+    | None -> ()
+    end;
+    host
+  in
+
+  let setup_test () =
+    let __context = Mock.make_context_with_new_db "Mock context" in
+    let session_id = Test_common.make_session ~__context () in
+    let __context = Context.make ~__context ~session_id "test" in
+    let make_host = make_host __context in
+
+    let sr_of_vdi = Test_common.make_sr ~__context () in
```

Comment:
Slightly confusing name: `sr_of_vdi` looks as if it should be a function.
`sr_for_vdi` would be clearer, at least for me.